src/lib/moduleResolver.ts

```diff
@@ -14,18 +15,30 @@ export interface ToAbsMid {
 	(moduleId: string): string;
 }
 
+export const RESOLVE_CONTENTS = Symbol();
+
 export default function makeResolver(toAbsMid: ToAbsMid): ResolveMid {
-	return function resolveMid<T>(mid: string): Promise<T> {
+	return function resolveMid<T>(mid: string, member: string | symbol = 'default'): Promise<T> {
 		return new Promise((resolve) => {
 			// Assumes require() is an AMD loader!
-			require([toAbsMid(mid)], (module) => {
-				if (module.__esModule) {
-					resolve(module.default);
-				}
-				else {
-					resolve(module);
+			require([toAbsMid(mid)], resolve);
```

Comment:
Should we consider some sort of AMD loader detection and fail gracefully if not support... We might want to also structure this in a way that it is possible that we might environmentally detect other loaders in the future and attempt to code path for resolution there.